src/main/java/util/GitHubURL.java

```diff
@@ -49,7 +49,7 @@ public static boolean isUrlIssue(String url) {
     }
     
     public static boolean isPullRequestLoaded(String url)  {
-        return (url.endsWith("/commits") || url.endsWith("/files"));
+        return url.matches("https://github.com/([^/]+)/([^/]+)/pull/([0-9]+)([/commits,/files]*)");
```

Comment:
When using regular expressions, use a comment to explain the intended meaning of the expression (e.g. 8 character strings ending with a number) and write test cases with a few examples of matching strings and boundary cases.